kable-core/src/androidMain/kotlin/Bluetooth.kt

```diff
@@ -15,75 +16,89 @@ import android.os.Build.VERSION.SDK_INT
 import android.os.Build.VERSION_CODES.R
 import androidx.core.content.ContextCompat
 import androidx.core.location.LocationManagerCompat
+import com.juul.kable.AvailabilityReason.AdapterNotAvailable
+import com.juul.kable.AvailabilityReason.LocationServicesDisabled
+import com.juul.kable.AvailabilityReason.Off
+import com.juul.kable.AvailabilityReason.TurningOff
+import com.juul.kable.AvailabilityReason.TurningOn
 import com.juul.kable.Bluetooth.Availability.Available
 import com.juul.kable.Bluetooth.Availability.Unavailable
-import com.juul.kable.Reason.LocationServicesDisabled
-import com.juul.kable.Reason.Off
-import com.juul.kable.Reason.TurningOff
-import com.juul.kable.Reason.TurningOn
 import com.juul.tuulbox.coroutines.flow.broadcastReceiverFlow
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.combine
 import kotlinx.coroutines.flow.distinctUntilChanged
+import kotlinx.coroutines.flow.emitAll
+import kotlinx.coroutines.flow.flow
 import kotlinx.coroutines.flow.flowOf
 import kotlinx.coroutines.flow.map
 import kotlinx.coroutines.flow.onStart
 import android.bluetooth.BluetoothAdapter.ACTION_STATE_CHANGED as BLUETOOTH_STATE_CHANGED
 
-public actual enum class Reason {
+public actual enum class AvailabilityReason {
```

Comment:
Maybe  in-availability reason? Seems like these are all reasons why it's inaccessible?

Reply:
Naming is hard. 😝 

I'll revert this actually, since "availability" is going away entirely soon (#737). So, this `enum` won't even be around for much longer anyways.